"npm run build" failing

Hey!

Hope you're doing well! I figured now that the README has some build directions I might give it a try, but running into an error on the last step. It seems to build `graphql-typer` without any issues, but then fails on a couple of the utils packages:

```
condict info build Matching packages: inflect, ipa, x-sampa
condict info build Running build command in matching packages
ERR! lerna Unknown arguments: @condict/ipa, @condict/x-sampa
condict info build Terminated by error: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @condict/workspace@ build: `node scripts/build -g build-tools && node scripts/build -g utils && node scripts/build -g server && node scripts/build -g ui`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @condict/workspace@ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I'll attach the full log file in case that helps:
[2020-09-13T15_06_53_811Z-debug.log](https://github.com/arimah/condict/files/5214379/2020-09-13T15_06_53_811Z-debug.log)

Let me know if there's any other info you need. Thanks!

--- a/scripts/build.js
+++ b/scripts/build.js
@@ -98,8 +98,7 @@ const main = () => {
     '--',
     // Arguments to lerna run in root
     '--loglevel=warn',
-    '--scope',
-    lernaScope,
+    `--scope=${lernaScope}`,
     // Arguments to npm run in each package
     '--',
     '-s'

